draft-ietf-ppm-dap.md

```diff
@@ -561,20 +560,19 @@ Aggregators. This scheme has two important properties:
   measurement from which it was generated.
 
 * Aggregators can compute secret shares of the aggregate result by aggregating
-  their shares locally into "aggregate shares", which may later be combined
-  into the aggregate result.
+  their shares locally into "aggregate shares", which may later be merged into
+  the aggregate result.
 
-Note that some VDAFs allow measurements to be aggregated multiple times, each
-time with a different aggregation parameter; however, DAP only allows each
-measurement to be aggregated once. Similarly, some VDAFs produce aggregate
-values which depend on the order in which the measurements are aggregated;
-however, DAP only supports VDAFs whose aggregation results are independent of
-the order in which measurements are aggregated (see {{Section 4.4.1 of !VDAF}}).
+DAP is not compatible with all VDAFs. Some VDAFs allow measurements to be
+aggregated multiple times with a different aggregation parameter. DAP only
+allows each measurement to be aggregated once. Some VDAFs produce aggregate
```

Comment:
At first I read this as "DAP doesn't support Mastic", which I don't think is the intent. It's true that there are certain things we can do with Poplar1/Mastic that we can't do in DAP, but there are still useful applications of Mastic (and Poplar1, kind of) that DAP supports.

Basically I would just avoid conflating multi-collection and order-sensitive VDAFs here.